graphics/gimp3-devel/Portfile

```diff
@@ -187,27 +198,32 @@ proc py_setup {py_ver} {
                     port:py${py_ver_nodot}-gobject3
 }
 
-variant python39 description {Build with python plugin support using python 3.9} {
+variant python39 conflicts python310 python311 python312 python313 description {Build with python plugin support using python 3.9} {
     py_setup 3.9
 }
 
-variant python310 description {Build with python plugin support using python 3.10} {
+variant python310 conflicts python39 python311 python312 python313 description {Build with python plugin support using python 3.10} {
     py_setup 3.10
 }
 
-variant python311 description {Build with python plugin support using python 3.11} {
+variant python311 conflicts python39 python310 python312 python313 description {Build with python plugin support using python 3.11} {
     py_setup 3.11
 }
 
-variant python312 description {Build with python plugin support using python 3.12} {
+variant python312 conflicts python39 python310 python311 python313 description {Build with python plugin support using python 3.12} {
     py_setup 3.12
 }
 
+variant python313 conflicts python39 python310 python311 python312 description {Build with python plugin support using python 3.13} {
+    py_setup 3.13
+}
```

Comment:
Now that Python 3.x can be used, the various python variants can be eliminated. That will be be fixed soon.

Reply:
I would love to stop having to worry about python versions throughout the build!